### PR TITLE
added printf and printint and printchar

### PR DESCRIPTION
**Pull request by imenyoo2**
added the following functions
- `printf`: only a prototype version, it's not complete
- `handleArg`: to handle printing the next argument (also not complete)
- `printint`: to print int types
- `printchar`: to print char types

### CODE REVIEW

**Comment by imenyoo2** [comment]
it seems that you also we need to refactor main branch to use `printchar` instead of `_putchar`

File: main.c [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#include "main.h"
+#include <stdarg.h>
```
Comment (SaaW008) [inline main.c]: the main.h already contain stdarg.h

File: printf.c [hunk 2]
```diff
@@ -0,0 +1,86 @@
+#include "main.h"
+#include <unistd.h>
+#include <stdarg.h>
+
```
Comment (SaaW008) [inline printf.c:2]: must contain the betty style

**Comment by imenyoo2** [comment]
i did some clearn up, it seems that `printf` is over 40 lines which betty doesn't like, let's just leave it this way until we see the checker output, maybe they'll ignore it

**Pull request merged**